### DISCUSSION TITLE
Facing Errors with using Cyfrin DevOpsTools library in Lesson 11

### DISCUSSION THREAD

**Discussion by subhradeepsamantray** [Cyfrin/foundry-full-course-cu#890] [Q&A] [answered]
Hello All,

I am getting errors while using the Cyfrin/DevOpsTools.sol library to get the most recent deployement for the PUG token. When running the "make mint ARGS="--network sepolia" command, I am getting the following error - 

![Screenshot (109)](https://github.com/Cyfrin/foundry-full-course-f23/assets/45929186/16c31ba3-7122-4c41-9925-c8579f625774)

![Screenshot (110)](https://github.com/Cyfrin/foundry-full-course-f23/assets/45929186/fa4f0d1d-79d4-498c-bb44-2527ca6ce81e)

It is showing "No Contract Deployed". I am not able to understand what is the issue. but I have first deployed the contract and then trying to mint it. I have followed all the steps mentioned by Patrick in the lesson and checked several times now. But not able to figure out the problem.

Please help if anyone else has faced the same issue in the past.

Thanks in advance 🙂

**Comment by shikhar229169** [comment, marked as answer]
You need to install jq.
```
sudo apt install jq
```

**Reply by subhradeepsamantray** [reply]
Hey @shikhar229169 , Thanks for the help, it worked!!

But can you explain what is this that we have to install and why it was required to install this package ?

**Reply by shikhar229169** [reply]
You can read about it [here](https://www.scaler.com/topics/linux-jq/)

When we deploy our smart contracts, the transaction information is saved in broadcast folder which contains the respective json file for every deployments, so jq allows to extract data from the required json file, and here we are extracting the contract address.

**Reply by subhradeepsamantray** [reply]
Thanks for the explanation !!
